docs/HowToUseMPU.md

```diff
@@ -1,9 +1,15 @@
 ## MPU
-* It supports 12 or 16 memory regions, depending on the configuration(ARMv7-R has 12 memory regions).
-* Each region should be programmed with a base address and size.
 
-### MPU Initialization and Memory Region Setup
-Initialization of MPU is done by setting up memory regions. Memory regions are set up by using the functions in below table and each function requires 2 arguments which are _Region Base Address_ and _Region Size_ to setup a memory region.
+## Contents
+- [MPU Initialization and Memory Region Setup](#mpu-initialization-and-memory-region-setup)
+- [Memory Region Setup Functions](#memory-region-setup-functions)
+- [Example](#example)
+
+It supports 12 or 16 memory regions, depending on the configuration. The configuration is so chosen, considering that ARMv7-R has 12 memory regions.
+Each region should be programmed with a base address and size.
```

Comment:
How about locating those two lines under `MPU Initialization and Memory Region Setup`? This looks strange.

Reply:
I changed according to the suggestion. However, if you consider placing it under `MPU Initialization and Memory Region Setup`, then I will change accordingly. Thank you.